Mage.Sets/src/mage/cards/c/CanalDredger.java

```diff
@@ -28,14 +26,7 @@ public CanalDredger(UUID ownerId, CardSetInfo setInfo) {
         this.power = new MageInt(1);
         this.toughness = new MageInt(5);
 
-        // TODO: Draft specific abilities not implemented
-        // Draft Canal Dredger face up.
-        this.addAbility(new SimpleStaticAbility(Zone.ALL, new InfoEffect("Draft Canal Dredger face up.")));
-
-        // Each player passes the last card from each booster pack to a player who drafted a card named Canal Dredger.
```

Comment:
please include/keep all rules text comments (no need for info effects)

Reply:
My apologies, I didn't realise Canal Dredger and Deal Broker were already in - I thought I was adding them as new files. Should revert to the existing files for those two.

And I am guessing you want the SimpleStaticAbility added to the others.

Another problem is I added the draftmatters cards at the bottom of the list of cards, whereas Canal Dredger and Deal Broker were added alphabetically, so are currently duplicated. I guess they should all be alphabetically - I will fix this now.

Reply:
Rather than removing info effects, it might be better for validation purposes and matching text to add a hint explaining that the draft effect won’t work